### PR TITLE
Split Travis into one job per coverage group

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
@Hexcles this builds on https://github.com/GoogleChromeLabs/confluence/pull/401 which I self-reviewed.

**Comment by foolip** [comment]
The lack of codecov comment on this PR is concerning. The push build finished before I created the PR, but maybe I've also broken codecov integration. Forcing an update to see what will happen.

**Comment by foolip** [comment]
Alright, seems fine, was probably a codecov infra issue.

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+    - npm run coverageNode
+
+  - name: firefox
+    node_js: 10
+    addons:
+      firefox: latest
+    before_script:
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+    - sleep 3 # give xvfb some time to start
```
Comment (Hexcles) [inline .travis.yml]: This makes me a bit uneasy, but I see this is the recommended way on Trusty (https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-directly).

Comment (foolip) [reply .travis.yml]: It is pretty unsightly, using xvfb-run might be an alternative, but then I'd rather look into running headless Chrome and Firefox somehow. So I'll leave it be :)

Comment (foolip) [reply .travis.yml]: Gave it a try: https://github.com/GoogleChromeLabs/confluence/pull/405

**Pull request merged**